### PR TITLE
Tidying up the newsletter guidelines

### PR DESCRIPTION
**Pull request by bethaitman**
Couple of tiny tidying things: mainly fixing a list that wasn't rendering properly and a broken link. Also fixed a typo elsewhere that was bugging me.

I _think_ I've got the right link syntax but haven't yet set things up on my machine to check it.

### CODE REVIEW

**Comment by plaindocs** [comment]
Hey @bethaitman you can preview changes in the Netlify preview: https://deploy-preview-715--dentist-canary-64055.netlify.com/

File: docs/organizer-guide/newsletter/newsletter-process.rst [hunk 1]
```diff
@@ -24,32 +25,32 @@ Assign stories (7-10 days before shipping)
...
 Assemble & review the newsletter (2-3 days before shipping)
-****************************************************************
+************************************************************
 
 Once all the story drafts have come in, it's time to assemble. They'll start with the ``newsletter-TEMPLATE-2017.rst`` document, and start to fill it in.
 
 The top section is an Editor's letter, which just serves as a quick greeting and introduction, and touches on any big things happening in the community during the month. This usually clocks in at 100-150 words.
 
-Next comes the stories. They should be edited for length and clarity and tone, as well as to make sure they've followed the `editorial guidelines, below <#editorial-guidelines>`_.
+Next comes the stories. They should be edited for length and clarity and tone, as well as to make sure they've followed the :doc:`newsletter/editorial-guielines`.
```
Comment (plaindocs) [inline docs/organizer-guide/newsletter/newsletter-process.rst]: :doc:`newsletter/editorial-guidelines`

**Review by plaindocs — APPROVED** [review]
Looks great - one typo.

File: docs/organizer-guide/newsletter/newsletter-process.rst (hunk 1 above)
Comment (bethaitman) [reply docs/organizer-guide/newsletter/newsletter-process.rst]: thanks, fixed!

**Pull request merged**